engine/src/options.h

```diff
@@ -41,52 +41,39 @@ class vs_options {
     void init();
 
 /* General Options */
-    bool write_savegame_on_exit;
-    int times_to_show_help_screen;
-    bool remember_savegame;
-    std::string new_game_save_name;
-    int quick_savegame_summaries_buffer;
-    std::string empty_mission;
-    std::string custompython;
-    bool quick_savegame_summaries;
-    int garbagecollectfrequency;
-    uint numoldsystems;
-    bool deleteoldsystems;
-    char vsdebug;
-    bool while_loading_starsystem;
     std::string default_mission;
     std::string galaxy;
-    bool command_interpretor;
-    bool load_last_savegame;
+    bool command_interpretor{};
```

Comment:
okay - so I know I gave @royfalk some grief previously about using this format vs a more normal legacy method, and I'm going to take the moment to say that at the time that was the right call; but now I think it's a good time to start using this format more. We've had a couple releases done and established and it's slowly becoming more of our - your - own code.

I'd also like to call out that this will probably be the first of many rounds like this. Overall we'll keep changes this way a bit more conservative and as you all get us older folk use to reading it we'll start allowing more and more of it in as it makes sense.

@royfalk I haven't forgotten ;) just want to call out the above and not let it seem like I'm letting is slide under the radar. I've noticed.

Reply:
I'll be happy to be able to use this format more going forward. ;-)